client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.html

```diff
@@ -28,6 +28,16 @@
                         </button>
                     </li>
                     <li class="divider" role="presentation"></li>
+                    <li role="menuitem">
+                        <a class="btn fs14 text-left"
+                           data-ui-sref="iaas.pci-project.compute.infrastructure.iac-view()"
```

Comment:
I guess there's no need for parentheses there. Isn't this supposed to be a reference to a state, not a function?

Reply:
It was inspired by other part on this file.